src/main/resources/templates/xbox.html

```diff
@@ -2,11 +2,11 @@
 <html lang="en" xmlns:th="http://thymeleaf.org">
 <head>
 <meta charset="ISO-8859-1">
-<!-- Required meta tags -->
+ 
 <meta charset="utf-8">
 <meta name="viewport"
 	content="width=device-width, initial-scale=1, shrink-to-fit=no">
-<!-- Bootstrap CSS -->
+
```

Comment:
Removed unnecessary comment and also deleted the start.html file as it was unnecessary. I believe you copied our boilerplate code from class